superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/DatasetPanel.tsx

```diff
@@ -168,15 +191,48 @@ export interface IDatasetPanelProps {
    * Boolean indicating if the component is in a loading state
    */
   loading: boolean;
+  linkedDatasets?: DatasetObject[] | undefined;
 }
 
+const renderExistingDatasetAlert = (linkedDataset: any) => (
```

Comment:
Can we put a type on this linkeddataset?

Reply:
Oops, good catch! Typed in [`this commit`](https://github.com/apache/superset/pull/22136/commits/86549660bd4d6ac53e0b7723ab3e9370e37d2a28#diff-544198bdd029b6e92c7a16fd435544ef915ae29e2de080abc82b23377094138eR213).